ManagedCode.Storage.Aws/Options/AWSStorageOptions.cs

```diff
@@ -3,12 +3,49 @@
 
 namespace ManagedCode.Storage.Aws.Options;
 
+/// <summary>
+///     Configuration options for AWS S3 storage.
```

Comment:
Personally I don't use indentation for the XML comments, but I saw that this was used other places in this repository.

Reply:
I do this in copilot, so no worrirs :)